_all.pcss

```diff
@@ -9,6 +9,7 @@
 
 /* Variables */
 @import "variables/_box-shadows.pcss";
+@import "variables/_layers.pcss";
```

Comment:
Let's update this file name to be a bit more clear on what it is `z-index.pcss` ; )

Reply:
+1 to this.